debian.sh

```diff
@@ -18,42 +20,41 @@ setup_network_config() {
     else
       UDEVFILE="/dev/null"
     fi
-    echo -e "### Hetzner Online GmbH - installimage" > $UDEVFILE
-    echo -e "# device: $1" >> $UDEVFILE
-    echo -e "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", ATTR{address}==\"$2\", ATTR{dev_id}==\"0x0\", ATTR{type}==\"1\", KERNEL==\"eth*\", NAME=\"$1\"" >> $UDEVFILE
-
-    echo -e "### Hetzner Online GmbH - installimage" > $CONFIGFILE
-    echo -e "# Loopback device:" >> $CONFIGFILE
-    echo -e "auto lo" >> $CONFIGFILE
-    echo -e "iface lo inet loopback" >> $CONFIGFILE
-    echo -e "" >> $CONFIGFILE
-
-    if [ "$3" -a "$4" -a "$5" -a "$6" -a "$7" ]; then
-      echo -e "# device: $1" >> $CONFIGFILE
-      echo -e "auto  $1" >> $CONFIGFILE
-      echo -e "iface $1 inet static" >> $CONFIGFILE
-      echo -e "  address   $3" >> $CONFIGFILE
-      echo -e "  netmask   $5" >> $CONFIGFILE
-      echo -e "  gateway   $6" >> $CONFIGFILE
+    echo "### $COMPANY - installimage" > "$UDEVFILE"
```

Comment:
https://github.com/virtapi/installimage/issues/49 for the udev part. I will update the interface file handling to a few printf